exchanges/exmo/exmo_wrapper.go

```diff
@@ -177,16 +177,29 @@ func (e *EXMO) FetchTradablePairs(ctx context.Context, a asset.Item) (currency.P
 		return nil, err
 	}
 
-	pairs := make([]currency.Pair, len(symbols))
-	var target int
+	// This secondary call is required to get last updated time for each pair.
+	// Some pairs are not currently supported by the API but are not listed as
+	// such in the first call.
+	tickers, err := e.GetTicker(ctx)
+	if err != nil {
+		return nil, err
+	}
+
+	pairs := make([]currency.Pair, 0, len(symbols))
 	for key := range symbols {
+		info, ok := tickers[key]
+		if !ok {
+			return nil, errors.New("could not match pair to ticker info")
+		}
+		if time.Since(time.Unix(info.Updated, 0)) >= time.Hour {
```

Comment:
What about illiquid pairs which trade infrequently? IMO this should be filtered by the calling application instead